### ISSUE TITLE
How to use BraceWrapping

### ISSUE THREAD

**Issue by alecive** [rosshemsley/SublimeClangFormat#45]
Hello, 

thanks for this very useful tool! I have a question related to `bracewrapping`, because I cannot make it work. A standard `clang-format` file should have something like this:

```
BraceWrapping:
  AfterClass:      false
  AfterControlStatement: false
  AfterEnum:       false
  AfterFunction:   true
  AfterNamespace:  false
  AfterObjCDeclaration: false
  AfterStruct:     false
  AfterUnion:      false
  BeforeCatch:     false
  BeforeElse:      false
  IndentBraces: false
```

How to replicate this in a custom style for this package?

**Comment by zorg93** [comment]
I have created a PR( #49 ) with support for this.

**Comment by rapimo** [comment]
closed by #49

**Comment by adsherman** [comment]
Hi! Something about this update has broken my ability to format. My clang_format_custom.sublime-settings (Clang format default) reflects the update, but when I go to actually format I get the error:

Clang format: YAML:1:254: error: unknown key 'BraceWrapping'
{AlignTrailingComments: true, AllowShortLoopsOnASingleLine: true, BasedOnStyle: LLVM, ColumnLimit: 150, AllowShortIfStatementsOnASingleLine: true, AllowAllParametersOfDeclarationOnNextLine: false, BinPackArguments: false, UseTab: Never, BraceWrapping: {}, TabWidth: 4, IndentWidth: 4}
                                                                                                                                                                                                                                                             ^
Error parsing -style: Invalid argument
